### ISSUE TITLE
Wordpress 5.6 Compatibility

### ISSUE THREAD

**Issue by MattMski** [onigetoc/Zajax-Wordpress-ajax-navigation#3]
Hello, I am trying to use this on the new WordPress update but it is broken as WordPress has updated their jQuery version.
Will this plugin be updated for jQuery 3?

**Comment by FabLabSNCF** [comment]
hi! with JQueryManager, we can downgrade JQuery on Wordpress to JQuery 1, and make Zajax operationnal.
But it's not a solution for the future. Please help us to make Zajax compatible with JQuery 3! 

Thanks :)

----------------------

Bonjour ! Avec JQueryManager, il est possible de réactiver la version JQuery 1 sur Wordpress, et faire fonctionner Zajax.
Mais ce n'est pas une solution stable ! Pouvez-vous nous aider à rendre Zajax compatible avec JQuery 3 ?

Merci :)

**Comment by onigetoc** [comment]
Hi,
I just fix and update Zajaz pro and it's seem to work well on my side.

Go to your download from menu "My Account" and choose download.
You can upload all file or just unzip and change the file zajax.js in the
js folder.

You can deactivate the jQuery Migrate Helper plugin.
Empty all cache. Sometime the javascript file are still in the browser
cache.

What i usually do it's to open the javascript file in the browser and hit
F5 on my keyboard (this do a browser cache clean up)
the javascript file should be here.
https://my-website-url.com/wp-content/plugins/zajax-pro/js/zajax.js

Best regard
Gino
Scriptsmashup.com



Le lun. 21 déc. 2020 à 02:42, FabLabSNCF <notifications@github.com> a
écrit :

**Comment by MattMski** [comment]
Will it be updated for the free version?

**Comment by onigetoc** [comment]
Hi, this url is a example.
You should use your url after updating the plugin.
Did you follow and read the procedure.

Update Zajax first and refresh the javascript to clear browser cache.
You should replace my-website-url by your own url.

https://www.scriptsmashup.com/my-account

Regard
Gino

**Comment by MattMski** [comment]
Yeah I had understood that wrong in the beginning, my bad. But I am using the free version so I was wondering if that version was going to be updated.
